### PR TITLE
Add/delete experiment functionality and basic filtering are here

### PR DESCRIPTION
**Pull request by yvikhlya**
Test url: https://gmao.gsfc.nasa.gov/intranet/research/modeling/agcm/geos5/yury/GEOS_Web/index.php

Experiment list is kept in sqlite database, which can be modified only if database file and containing directory have "write all" permissions. This is a security "swiss cheese" (a lot of big holes). Our sysadmins should help us to configure the server properly, such that users could access the database using their launchpad login credentials.

Experiments list is incomplete, and there are other bits of functionality missing, and some code can be optimized, which will be easy to fix once the major issues are resolved.

### CODE REVIEW

**Comment by mathomp4** [comment]
@yvikhlya Very nice!

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,13 +7,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Added
+### Added basic functionality including add/delete experiment and basic filtering
```
Comment (mathomp4) [inline CHANGELOG.md]: @yvikhlya Could you actually add this under `### Added` a la:

### Added

- Basic functionality ...

Comment (yvikhlya) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,13 +7,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Added
+### Added 
+- Basic functionality including add/delete experiment and basic filtering
```
Comment (yvikhlya) [inline CHANGELOG.md:11]: Resolved

**Review by mathomp4 — APPROVED** [review]

**Comment by mathomp4** [comment]
Merged!

**Pull request merged**